### DISCUSSION TITLE
Adding folders to lively

### DISCUSSION THREAD

**Discussion by ghost** [rocksdanister/lively#452] [Ideas]
Hey,
Yesterday I was adding new wallpapers to lively and thought that it could be a good feature to add folders to lively to organize better all my wallpapers. 
This is just a proposition, if you like it it would be a great idea to add it 😄.

**Comment by rocksdanister** [comment]
This is not possible to implement in an elegant way, html and application wallpapers can have their own folder with media contents.. so will be impossible to distinguish between them and standalone media files.

Also next update will have batch file import; Just Ctrl + A  all the files:
https://www.youtube.com/watch?v=Nk6sF1r9J0U

Beta is already out:
https://github.com/rocksdanister/lively-beta/releases

**Reply by ghost** [reply]
ok, thank you for replying!
